### ISSUE TITLE
[1.3-modules-beta.2] Unable to use wildcard for export

### ISSUE THREAD

**Issue by xamfoo** [meteor/meteor#5897]
Tried:
```javascript
export * from './validateEmail'
```
Error when compiling:
```
TypeError: Object [object Object] has no method 'interopExportWildcard'
    at meteorInstall.node_modules.caloric:util.server.js (packages/caloric:util/server.js:1:32)
    at fileEvaluate (packages/modules/.npm/package/node_modules/install/install.js:183:1)
    at require (packages/modules/.npm/package/node_modules/install/install.js:75:1)
    at /home/xam/src/.meteor/local/build/programs/server/packages/caloric_util.js:60:15
    at /home/xam/src/.meteor/local/build/programs/server/packages/caloric_util.js:66:3
    at /home/xam/src/.meteor/local/build/programs/server/boot.js:242:10
    at Array.forEach (native)
    at Function._.each._.forEach (/home/xam/.meteor/packages/meteor-tool/.1.1.11-modules.3.wwtctz++os.linux.x86_64+web.browser+web.cordova/mt-os.linux.x86_64/dev_bundle/server-lib/node_modules/underscore/underscore.js:79:11)
    at /home/xam/src/.meteor/local/build/programs/server/boot.js:137:5
```

A workaround is to change it to `export {validateEmail} from './validateEmail'` but this can get verbose and error-prone if there are multiple properties.

**Comment by trusktr** [comment]
Dupe of #5872

**Comment by stubailo** [comment]
Yeah, and I think you meant `import` right?

**Comment by trusktr** [comment]
@stubailo That should also [work with exports](http://www.2ality.com/2014/09/es6-modules-final.html#re-exporting).

**Comment by stubailo** [comment]
Oh wow, had no idea.

**Comment by trusktr** [comment]
@stubailo There's also a nice proposal to extend `exports` to match all the semantics of `import`: https://github.com/leebyron/ecmascript-more-export-from

**Comment by xamfoo** [comment]
@stubailo I specifically mean `export` here.

This is different from `import` in #5872.

`export * from './file'` is valid syntax in E2015. It re-exports the items from another file just like `export {something} from './file'`.

See:
http://www.ecma-international.org/ecma-262/6.0/#sec-module-namespace-exotic-objects
http://www.2ality.com/2014/09/es6-modules-final.html#re-exporting

**Comment by mquandalle** [comment]
I’m also experiencing this bug, I think this issue should be re-open.

**Comment by stubailo** [comment]
I think @benjamn is the best to handle it.

**Comment by avital** [comment]
Fixed with 3a9907a. Will be out in the next beta.
